widgets/src/mocks/app-builder/widgets.js

```diff
@@ -11,7 +11,7 @@ export default {
     taskId: '290',
     pageCode: 'phase_1_widgets',
     frameId: '4',
-    widgetCode: 'phase_1_widgets_task_details',
+    widgetCode: 'task_details',
```

Comment:
why are you changing this?

Reply:
sorry this was unintended, I was testing most existing widgets. I reverted it now